### ISSUE TITLE
Reactors Duplicate Yellorium

### ISSUE THREAD

**Issue by Sinazok** [erogenousbeef/BigReactors#565]
Reactors with greater than 9000 mB available fuel capacity that is not divisible by 9000 will duplicate yellorium when added as blocks in excess of said available capacity. Tested using Big Reactors 0.4.3A for Minecraft 1.7.10.

To reproduce this bug, create a reactor with exterior dimensions 4x4x3 using 4 yellorium fuel rods. Place 2 yellorium blocks into the inlet of an access port at the same time. The reactor will take 1 block and leave the other, but the internal fuel will read 16000. You can then eject the fuel and acquire 16 ingots instead of 9. This can be repeated as much as you like. A stack of 64 yellorium blocks produced 113 blocks worth of ingots in return.

**Comment by CristinoValdigem** [comment]
Take a Screenshot based on model you sad... I think the explanation is better

**Comment by Sinazok** [comment]
Here is a link to screenshots detailing the duplication process.
http://kozynetwork.net/YelloriumBug

**Comment by CristinoValdigem** [comment]
That is crazy o.O

**Comment by Sinazok** [comment]
I actually discovered this while testing to make sure reactors were not wasting yellorium by taking whole blocks to top-up. Realising the game-breaking implications when I found the exact opposite to be true, I reported it immediately. Hopefully this can be resolved without causing the issue I originally hoped not to find.

**Comment by Tomson124** [comment]
Hey @Sinazok, as erogenousbeef doesn't have time anymore to maintain BigReactors some guys and me "took over" the mod. Would be good if you could open this issue at our repo, too.
https://github.com/CyberdyneCC/BigReactors
Of course, this is a real problem and I will look into it.

**Comment by CristinoValdigem** [comment]
This Guy will continue developing Big Reactors ?? @Tomson124

**Comment by Tomson124** [comment]
Yeah, CyberdyneCC is a group of people including myself, who will continue the work on BigReactors.

**Comment by CristinoValdigem** [comment]
@Tomson124 One question... The group take the issue reports from what page ?? That or CyberdyneCC ?? Because i've report a issue months ago about a bug with mekanism cables

**Comment by sameer** [comment]
CyberdyneCC @CristinoValdigem
